### PR TITLE
Prepare a codeowners file for github actions

### PR DESCRIPTION
**Pull request by frenzyfriday**

### CODE REVIEW

**Comment by frenzyfriday** [comment]
The usernames are wrong -I will update them soon

**Review by lewisdenny — CHANGES_REQUESTED** [review]
Hi @frenzyfriday, 

Some extra suggestions:

@vakwetu for roles/compliance
@hjensas for roles/nat64_appliance and roles/libvirt_manager and roles/ssh_jumper and roles/sushy_emulator
@fmount  for roles/os_must_gather
@dpinhas  for roles/ovirt
@kopecmartin for roles/polarion
@jirimacku for roles/reportportal
@sathlan @Jaganathancse for roles/update

NFV: roles/tofu

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+roles/ci_lvms_storage @fultonj @fmount @abays
+roles/ci_local_storage @fultonj @fmount @abays
+
+# DCN
+roles/ci_dcn_site @fultonj @sbekkerm @krcmarik
+playbooks/dcn.yml @fultonj @sbekkerm @krcmarik
+
+# Polarion
+roles/polarion @tosky @ciecierski
```
Comment (tosky) [inline .github/CODEOWNERS:20]: jparoly and jirimacku as primary

**Review by tosky — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (frenzyfriday) [reply .github/CODEOWNERS]: Thanks, added!

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+roles/os_must_gather @fmount
+
+# NFV
+roles/idrac_configuration @eshulman2 @rdiazcam @mnietoji
+roles/ovirt @eshulman2 @rdiazcam @mnietoji
+roles/switch_config @eshulman2 @rdiazcam @mnietoji
+roles/tofu
+
+# Ovirt
+roles/ovirt @dpinhas
```
Comment (dpinhas) [inline .github/CODEOWNERS:24]: I believe RH employee should be code-owner of this role :)

**Review by dpinhas — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (frenzyfriday) [reply .github/CODEOWNERS:24]: oh shoot :D

**Comment by frenzyfriday** [comment]
> Hi @frenzyfriday,
> 
> Some extra suggestions:
> 
> @vakwetu for roles/compliance @hjensas for roles/nat64_appliance and roles/libvirt_manager and roles/ssh_jumper and roles/sushy_emulator @fmount for roles/os_must_gather @dpinhas for roles/ovirt @kopecmartin for roles/polarion @jirimacku for roles/reportportal @sathlan @Jaganathancse for roles/update
> 
> NFV: roles/tofu

Martin is leaving openstack right?

**Comment by lewisdenny** [comment]
> > Hi @frenzyfriday,
> > Some extra suggestions:
> > @vakwetu for roles/compliance @hjensas for roles/nat64_appliance and roles/libvirt_manager and roles/ssh_jumper and roles/sushy_emulator @fmount for roles/os_must_gather @dpinhas for roles/ovirt @kopecmartin for roles/polarion @jirimacku for roles/reportportal @sathlan @Jaganathancse for roles/update
> > NFV: roles/tofu
> 
> Martin is leaving openstack right?

True, ignore that one, I just parsed the git history and forgot about that move!

File: .github/CODEOWNERS (hunk 2 above)
Comment (lewisdenny) [reply .github/CODEOWNERS:24]: It's opensource mate, get amongst it :D

Comment (dpinhas) [reply .github/CODEOWNERS:24]: @lewisdenny Of course! Seriously I'll be happy to contribute, as long as I'm pinged/mentioned in GitHib issues.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# Must gather
+roles/os_must_gather @fmount
+
+# NFV
+roles/idrac_configuration @eshulman2 @rdiazcam @mnietoji
+roles/ovirt @eshulman2 @rdiazcam @mnietoji
+roles/switch_config @eshulman2 @rdiazcam @mnietoji
+roles/tofu
+
+# Ovirt
```
Comment (lewisdenny) [inline .github/CODEOWNERS:23]: We will need to drop Ovirt and support it ourselves, I don't know anyone else to put here for now. 

Once done, /lgtm

**Review by lewisdenny — CHANGES_REQUESTED** [review]
^

File: .github/CODEOWNERS (hunk 3 above)
Comment (frenzyfriday) [reply .github/CODEOWNERS]: ack, removed

**Review by lewisdenny — CHANGES_REQUESTED** [review]

**Review by lewisdenny — APPROVED** [review]
/approve

**Comment by pablintino** [comment]
@frenzyfriday as I mentioned privately, can we use groups? If the group has write permissions and it is specified using the fully qualified name, GH perfectly takes them.

**Comment by frenzyfriday** [comment]
> @frenzyfriday as I mentioned privately, can we use groups? If the group has write permissions and it is specified using the fully qualified name, GH perfectly takes them.

Ack, lemme try that. It did not work with the adoption group - maybe they were just not added to the repo. Lemme check

**Review by lewisdenny — COMMENTED** [review]
I already approved but the patch additions are nice, thanks for updating to using groups.

**Pull request merged**